sdk/core/core-http/samples/node-sample.ts

```diff
@@ -18,13 +18,13 @@ const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"] || "subscriptionId";
 // 1.5 in the curl tab you will see the actual curl request that has the bearer token in it
 // 1.6 copy paste that token here. That token is valid for 1 hour
 const token = process.env["ACCESS_TOKEN"] || "token";
-const creds = new msRest.TokenCredentials(token);
```

Comment:
`TokenCredentials` still exists as a way for someone to create a class that follows the interface `ServiceClientCredentials`. I believe this would send mixed messages on what our library officially supports.

Have we considered removing the `TokenCredentials` class?

Reply:
I'll see where it's being used and try to remove it if it seems safe.

Reply:
Removed it in my latest changes, also searched around and didn't find any direct uses of it in our SDK code.  Replaced it with `RawTokenCredential` wherever it was used in samples or tests.